apps/core/scenes/resultDetail/resultDetailContent/ResultDetailPassenger.js

```diff
@@ -79,16 +82,18 @@ class ResultDetailPassenger extends React.Component<Props, State> {
   state = {
     isFormVisible: false,
     passengerData: defaultPassengers,
+    currentEditID: '',
   };
 
   handleFormSaveRequest = passenger => {
     const existsID: boolean = !!this.state.passengerData.find(
       el => el.id === passenger.id,
     );
+
     const validateForm = () => {
       if (!(passenger.id && passenger.name)) {
         return 'ID and Name are mandatory';
-      } else if (existsID) {
+      } else if (existsID && this.state.currentEditID !== passenger.id) {
```

Comment:
```suggestion
      }  
      if (existsID && this.state.currentEditID !== passenger.id) {
```

I suppose that the `else` is not necessary.